src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/SelectVariants.java

```diff
@@ -1211,4 +1244,14 @@ private boolean needNumFilteredGenotypes(){
                 maxFractionFilteredGenotypes != MAX_FRACTION_FILTERED_GENOTYPES_DEFAULT_VALUE ||
                 minFractionFilteredGenotypes != MIN_FRACTION_FILTERED_GENOTYPES_DEFAULT_VALUE;
     }
+
+    /**
+     * Need the number of no-call genotyped samples?
```

Comment:
This comment is somehow confusing to me.  I think it would read better as: "Should the number of no-call genotypes be considered for filtering"